pombola/settings/south_africa.py

```diff
@@ -4,12 +4,17 @@
 from .base import *  # noqa
 from .south_africa_base import *  # noqa
 
+from pombola.south_africa.search import ZAElasticBackend
 
 HAYSTACK_CONNECTIONS['default']['EXCLUDED_INDEXES'] = [
     'pombola.search.search_indexes.PlaceIndex',
     'speeches.search_indexes.SpeechIndex',
 ]
 
+# Change to custom backend for
```

Comment:
Just half a comment I think. I probably intended to remove all of it. Well spotted though!